README.md

```diff
@@ -1,14 +1,48 @@
-`svgdigitizer.py` can be used to generate xy data of any kind from svg files. It creates data from multiple paths in a single plot.  
+# SVGDigitizer — Create (x,y) Data Points from SVG files.
+
+![Logo](./logo.svg)
+
+The purpose of this project is to digitize plots in scientific publications, recovering the measured data visualized in such a plot.
+
+## Installation
+
+```
+pip install -e .

```

Comment:
The package is now installable with pip. This puts `svgdigitizer` on the path. You might have to open a new shell or type `hash -r` for `svgdigitizer` to actually work initially.